crates/oxc_estree/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #[cfg(feature = "serialize")]
-mod serialize;
+pub mod serialize;
+
```

Comment:
If we get rid of `Range` type, can revert this change, which will fix the clippy errors.